src/pages/SigningOfficialConsole.js

```diff
@@ -1,245 +1,149 @@
-import {Component} from 'react';
-import {a, div, h, hh, hr, span} from 'react-hyperscript-helpers';
-import {PageHeading} from '../components/PageHeading';
-import {PageSubHeading} from '../components/PageSubHeading';
-import {PaginatorBar} from '../components/PaginatorBar';
-import {SearchBox} from '../components/SearchBox';
-import {PendingCases} from '../libs/ajax';
-import {Storage} from '../libs/storage';
-import manageUsersIcon from '../images/icon_manage_users.png';
-import accessIcon from '../images/icon_access.png';
-export const SigningOfficialConsole = hh(class ChairConsole extends Component {
-
-  constructor(props) {
-    super(props);
-    this.state = {
-      showModal: false,
-      currentUser: {},
-      dulLimit: 5,
-      darLimit: 5,
-      currentDulPage: 1,
-      currentDarPage: 1,
-      totalDulPendingVotes: 0,
-      totalAccessPendingVotes: 0,
-      electionsList: {
-        dul: [],
-        access: []
+import {useState} from "react";
+import {useEffect} from "react";
+import {Notifications} from "../libs/utils";
+import {div, a, h} from "react-hyperscript-helpers";
+import {Theme, Styles} from "../libs/theme";
+import DarTableSkeletonLoader from "../components/TableSkeletonLoader";
+import {tableHeaderTemplate} from "../components/dar_table/DarTable";
+import {tableRowLoadingTemplate} from "../components/dar_table/DarTable";
+import {User} from "../libs/ajax";
+import {img} from "react-hyperscript-helpers";
+import lockIcon from "../images/lock-icon.png";
+import userIcon from "../images/icon_manage_users.png";
+import SearchBar from "../components/SearchBar";
+import {darSearchHandler} from "../libs/utils";
+import {userSearchHandler} from "../libs/utils";
+import {assign} from "lodash/fp";
+import {span} from "react-hyperscript-helpers";
+
+export default function SigningOfficialConsole(props) {
+  const [signingOfficial, setSiginingOfficial] = useState();
+  //to be used for the manage dar component
+  const [darList, setDarList] = useState();
+  const [filteredDarList, setFilteredDarList] = useState();
+  const [currentDarPage, setCurrentDarPage] = useState();
+  //to be used for manage researcher component
+  const [userList, setUserList] = useState();
+  const [filteredUserList, setFilteredUserList] = useState();
+  const [currentUserPage, setCurrentUserPage] = useState();
+  const [isLoading, setIsLoading] = useState(true);
+
+  //could benefit in moving this to utils or a shared component so it can be used for the adminManageUsers page
+  const widerColumnStyle = assign(Styles.TABLE.DATASET_CELL, {margin: '1rem 2%'});
+  const regWidthColumnStyle = assign(Styles.TABLE.SUBMISSION_DATE_CELL, {margin: '1rem 2%'});
```

Comment:
Since these are theming options, they could be in the table component itself. They're fine here for now.